.github/workflows/release-mockopampserver.yml

```diff
@@ -0,0 +1,94 @@
+# Release a tagged version of the '@elastic/mockopampserver' package.
+name: release-mockopampserver
+
+on:
+  push:
+    tags:
+      - mockopampserver-v*.*.*
+
+# 'id-token' perm needed for npm publishing with provenance (see
+# https://docs.npmjs.com/generating-provenance-statements#example-github-actions-workflow)
+permissions:
+  attestations: write
+  contents: write
+  packages: write
+  pull-requests: read
+  id-token: write
+
+jobs:
+  release:
+    runs-on: ubuntu-24.04
```

Comment:
As mentioned in the comment above the recommendation is to define here the write permission:

```
permissions:
  attestations: write
  contents: write
  packages: write
  pull-requests: read
  id-token: write
```

Reply:
commit f05ef3b